zboxcore/sdk/allocation.go

```diff
@@ -424,10 +424,11 @@ func (a *Allocation) StartChunkedUpload(workdir, localPath string,
 	}
 
 	if thumbnailPath != "" {
-		buf, err := os.ReadFile(fileName)
+		buf, err := sys.Files.ReadFile(thumbnailPath)
```

Comment:
what is the difference? and why not simple use `iotuil.ReadFile`

Reply:
it should be ported to cross-platforms. os.ReadFile or iotuil.ReadFile can't be done. eg in wasm, it is ported to memory system instead of file system